static_website/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "account" {
+}
+
+variable "region" {
+}
+
+variable "environment" {
+}
+
+variable "nubis_domain" {
+  default = "nubis.allizom.org"
+}
+
+variable "technical_owner" {
+  default = "infra-aws@mozilla.com"
+}
+
```

Comment:
This variable seems to serve 2 distinct purposes.

First, to name the site being hosted, creating the DNS records, for instance
Second, to name the backing bucket holding content.

We've learned the hard way that bucket naming must be globally unique, so the only reliable/repeatable way to achieve that is with UUID naming of the bucket themselves, with an optionally identifying prefix.

I think the parameter should more directly reflect the DNS name of the site, and that the bucket itself should be hidden inside this module. Returning its name as an output makes sense, to know where to push content to.